xbmc/filesystem/AddonsDirectory.cpp

```diff
@@ -62,12 +62,27 @@ const std::array<TYPE, 5> infoProviderTypes = {
   ADDON_SCRAPER_TVSHOWS,
 };
 
+const std::vector<TYPE> lookAndFeelTypes = {
+  ADDON_SKIN,
+  ADDON_SCREENSAVER,
+  ADDON_RESOURCE_IMAGES,
+  ADDON_RESOURCE_LANGUAGE,
+  ADDON_RESOURCE_UISOUNDS,
+  ADDON_VIZ,
+};
+
+
 
 static bool IsInfoProviderType(const AddonPtr& addon)
 {
   return std::find(infoProviderTypes.begin(), infoProviderTypes.end(), addon->Type()) != infoProviderTypes.end();
 }
 
+static bool IsLookAndFeelType(const AddonPtr& addon)
```

Comment:
Maybe rename this to `IsLookAndFeelTypeAddon()` and also provide a `IsLookAndFeelType()` which takes an addon type as a parameter. Then you can call `IsLookAndFeelType()` from `IsLookAndFeelTypeAddon()` and on line 198 and the naming is more accurate.